src/app/application/index.ts

```diff
@@ -48,6 +67,19 @@ export const services = (app: Record<string, Token>): ServiceDefinition[] => {
         app.components,
       ],
     }],
+
+    [token('application.routes'), {
+      service: routes,
+      arguments: [
+        $.notFoundView,
+      ],
+      labels: [
+        app.routes,
```

Comment:
For anything that is labelled, there's currently not a lot of point typing the token (ideally it would be typed using the type of the label anyway).

Plus we want to make sure these tokens can't be exported, i.e. they are kinda like "private tokens". If you need to overwrite one of these you'd need to re-expose it like it was before.

In the future, labels will themselves be inspectable, so you'll be able to get a specific token/service by inspecting the public label token.